src/ui/veles_mainwindow.cc

```diff
@@ -647,6 +702,21 @@ QTabBar* MainWindowWithDetachableDockWidgets::dockWidgetToTab(
   return nullptr;
 }
 
+void MainWindowWithDetachableDockWidgets::splitDockWidget2(
+    QDockWidget* first, QDockWidget* second, Qt::Orientation orientation) {
+#ifdef SPLIT_DOCK_WIDGET_ACTUALLY_WORKS_AS_ONE_COULD_EXPECT
```

Comment:
Of course it's not a built in define because QMainWindow::splitDockWidget doesn't work satisfactory on any platform.